operator/pkg/reconciliation/reconcile_reaper.go

```diff
@@ -0,0 +1,235 @@
+// Copyright DataStax, Inc.
+// Please see the included license file for details.
+
+package reconciliation
+
+import (
+	"fmt"
+	"github.com/datastax/cass-operator/operator/internal/result"
+	api "github.com/datastax/cass-operator/operator/pkg/apis/cassandra/v1beta1"
+	v1batch "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"math"
+	"strconv"
+)
+
+const (
+	ReaperUIPort             = 7080
+	ReaperAdminPort          = 7081
+	ReaperDefaultImage     = "thelastpickle/cassandra-reaper:2.0.5"
+	ReaperDefaultPullPolicy  = corev1.PullIfNotPresent
+	ReaperContainerName      = "reaper"
+	ReaperHealthCheckPath    = "/healthcheck"
+	ReaperKeyspace           = "reaper_db"
+	ReaperSchemaInitJob      = "ReaperSchemaInitJob"
+	ReaperSchemaInitJobImage = "jsanda/reaper-init-keyspace:latest"
```

Comment:
This probably needs to be updated? Where does this init-container live at present?

Reply:
It currently lives at https://github.com/jsanda/create_keyspace. It is deployed as a job, not as an init container. The schema has to be initialized in a job in order to prevent concurrent schema modifications.

Once the management API supports creating keyspaces and tables, this won't be necessary.

Reply:
>Have we run integration tests against this branch?

I have not run all integration tests locally. I just committed/pushed an integration that I just realized hadn't been committed.

Reply:
How do we build new versions of this image and deploy them? Should it be just another image built by the cass operator build process (we already have more than one image we build).

Reply:
That is a good question. I personally lean towards keeping it separate because I am using it for a stand-alone operator for Reaper as well. With that said, hopefully this can altogether replaced with calls to the mgmt API in the near future so I am open to whatever makes the most sense in the interim.

Reply:
I think leaving this as-is is fine for merging this PR